source/core/src/main/com/csse3200/game/ui/DialogueBox/DialogBox.java

```diff
@@ -2,28 +2,39 @@
 
 import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.scenes.scene2d.Actor;
 import com.badlogic.gdx.scenes.scene2d.Stage;
-import com.badlogic.gdx.scenes.scene2d.ui.Dialog;
-import com.badlogic.gdx.scenes.scene2d.ui.Label;
-import com.badlogic.gdx.scenes.scene2d.ui.Skin;
-import com.badlogic.gdx.scenes.scene2d.ui.Table;
-import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.ui.*;
 import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
-import com.csse3200.game.services.ServiceLocator;
 
 public class DialogBox extends Dialog {
```

Comment:
class name too generic (and shared with another deprecated package) and isn't a dialogbox as it is only used for the selection screen popup; NPC dialogue is handled with the ChatOverlay class in team 3/4 branches. Please rename class and package